data/help.txt

```diff
@@ -65,9 +68,26 @@ help "lost 7"
 help "map"
 	`This map shows all the star systems you know about and the hyperspace links between them. You can use the map to plot a hyperspace course by clicking on the system you want to travel to. To initiate a jump to that system, close the map and press <Initiate hyperspace jump>.`
 
+help "map advanced"
+	`As you explore more of the galaxy, the star map will quickly become a very powerful tool. Clicking on a distant system will automatically plot the quickest route to it. If you have missions spread across several systems, you can shift-click each to plot a route between them, or control-click a system to inspect missions there without setting a route.`
+	`There are also many other useful modes the map can be set to. Click on one of the buttons in the lower right corner to learn more.`
+
+help "map advanced ports"
+	`The Ports panel provides detailed information about every planet, station, or other stellar objects you've come across. Most information can be clicked on (like the word "Outfitter", or on the price of a trade good), which will color every system on the star map appropriately.`
+	`For instance, you can click on "(has been visited)" to see which planets you've visited or not, or click on a trade good price to show how prices change throughout the galaxy. If you have a route planned already, this can be combined with a control-click to inspect trade good prices at stops along the way. A key will be shown on the right to let you know what colors mean in any given mode.`
```

Comment:
You may need to break this up into two different help dialogs if this is too long. (After reading one, the second one would show.)
```suggestion
	`The ports panel shows you where systems with spaceports are, as well as a plethora of other information. The panel can be switched to multiple separate keys which each show different information about the systems you've visited.`
	`Clicking on the name of the system or the word under the name of any planets in the system on the left side of the screen will switch the map to the reputation key. This will show you which planets are friendly, hostile, or have restricted landing access in each system. Note that if a system has any friendly planet in it, then the system will show as friendly, so be careful about friendly planets in potentially hostile regions of space.`
	`Clicking on the government of the system will switch the map to the government key, highlighting inhabited system according to which government owns them.`
	`Clicking the name of a planet, or clicking on a planet in the orbit map on the right side of the screen, will show that planet's description if you have visited it already. Clicking a planet on the orbit map will also tell your auto-pilot to land on that planet after you have arrived at that system at the end of your jump path.`
	`Clicking on the words "Shipyard" or "Outfitter" under a planet name will change the map key to show how many ships or outfits are for sale on each planet.`
	`Clicking on "(has been visited)" or "(not yet visited)" under a planet name will change the map to the visited planets key, showing the systems where you have visited all, some, or none of the planets.`
	`Clicking on any trade commodity will use the trade key, highlighting the map according to the prices of that commodity in each system.`
```

Reply:
The problem with the Ports panel isn't that every clickable text isn't fully enumerated, it's that people don't know you can click anything at _all_. By encouraging the player to experiment with clicking on different elements, combined with pointing out the color key should get most people through it, while drastically cutting down on text and button presses. I definitely want to avoid multiple pages as well, since that can frustrating and overwhelming trying to close a prompt, only for another to appear, then _another_ to appear, especially when it's line after line of "Clicking on <text> will...".

Pointing out the interactive system map is definitely some major undocumented functionality, though. I'm going to try and see if I can neatly fit that in with something similar to the original. If not, or if you feel otherwise, we'll go with the multiple pages.

Reply:
As I said, I'd rather these be overly thorough than too vague. If we want to cut down on the wordiness of this, then instead of describing the result of clicking on each UI element, we simply describe each UI element that can be clicked on.
```suggestion
	`The ports panel shows you where systems with spaceports are, as well as a plethora of other information. The panel can be switched to multiple separate keys which each show different information about the systems you've visited.`
	 `Clicking on the name of the system, the government of the system, the lines under the name of each planet, or any trade commodity will change the map to use a different key, the purpose of which can be found on the right side of the screen.`
	 `Clicking the name of a planet, or clicking on a planet in the orbit map on the right side of the screen, will show that planet's description if you have visited it already. Clicking a planet on the orbit map will also tell your auto-pilot to land on that planet after you have arrived at that system at the end of your jump path.`
```

Reply:
That looks much better. Erring on the side of thoroughness is good, and these tutorials being delayed gives more wiggle room, but it's still a pretty careful balance to maintain.